src/plugins/formulas/formulas.js

```diff
@@ -20,6 +27,59 @@ export class Formulas extends BasePlugin {
     return PLUGIN_PRIORITY;
   }
 
+  /**
+   * Plugin settings.
+   *
+   * @private
+   */
+  #settings = this.hot.getSettings()[PLUGIN_KEY];
+
+  /**
+   * Flag used to bypass hooks in internal operations.
+   *
+   * @private
+   * @type {boolean}
+   */
+  #internal = false;
+
+  /**
+   * Flag needed to mark if Handsontable was initialized with no data.
+   * (Required to work around the fact, that Handsontable auto-generates sample data, when no data is provided).
+   *
+   * @type {boolean}
+   */
+  #emptyData = false;
+
+  /**
+   * Static register used to set up one global HyperFormula instance.
+   * TODO: currently used in tests, might be removed later.
+   *
+   * @private
+   * @type {object}
+   */
+  staticRegister = staticRegister('formulas');
+
+  /**
+   * The HyperFormula instance that will be used for this instance of Handsontable.
+   *
+   * @type {HyperFormula}
+   */
+  hyperformula = null;
+
+  /**
+   * HyperFormula's sheet name.
+   *
+   * @type {string}
+   */
+  sheetName = null;
+
+  /**
+   * HyperFormula's sheet id.
+   *
+   * @type {number}
+   */
+  sheetId = null;
```

Comment:
You're right 👍

Reply:
I'm curious why did you choose `sheetId` instead of `sheetName` to be the getter, if there's any specific reason? I think both would work fine, but it's a little odd.

Reply:
I didn't give it too much though, honestly. Most (?) HF methods require using `sheetName`, so I went with it.